### DISCUSSION TITLE
getConsentStatus() returns 3 regardless of declining or accepting consent

### DISCUSSION THREAD

**Discussion by MishaVersteeg** [admob-plus/admob-plus#449] [General]
To show non-personalized ads I have to pass ` npa="1"` when initializing the ads.

However, getConsentStatus() always returns 3 regardless of declining or accepting consent.

How can I know if a user has declined or accepted the consent?

**Comment by nosTa1337** [comment]
Same question here. Trying to figure this out for some time now how it works. It doesn't matter what I click, the status is always 3.

**Reply by TheNotorius0** [reply]
The npa system is **deprecated** now.

getConsentStatus will **always** return 3 after you obtain a consent (either accepted or declined)

0: Unknown
1: Required
2: NotRequired 
3: Obtained

You don't have to pass npa="1" anymore. AdMob will handle that for you (if the user has declined the consent it will show non-personalized ads)

It's not possibile for a developer to know the consent status of a single user anymore (I guess for the privacy?). The documentation isn't very clear about it, to be honest.
